.travis.yml

```diff
@@ -0,0 +1,14 @@
+branches:
+  except:
+    - master
+language: ruby
+cache: bundler
+sudo: false
+rvm:
```

Comment:
That's a good point, I think chef's embedded ruby is still 2.1.x.  I have chef-dk 0.13.21 that comes packaged with chef 12.9.41 and ruby 2.1.8. So I think this should definitely include ruby 2.1.<something>.